docker/python_utils/provenance.py

```diff
@@ -1,18 +1,27 @@
 #!/usr/bin/env python
 
 import json
+import os
 import synapseclient as sc
 import sys
+import urllib.request
 
-injson = sys.argv[1]
-synconf = sys.argv[2]
+# synapse config file
+synconf = sys.argv[1]
+#url to job.json file
+joburl = sys.argv[2]
+
+# Get the github url to the job.json file
+#joburl = os.getenv('CWL_ARGS_URL')
+
+#download the job.json file
+urllib.request.urlretrieve(joburl, 'job.json')
```

Comment:
what happens if github is down? I would put this in a try.

Reply:
Added error handling with requests